runtime/jvmMain/src/kotlinx/serialization/JvmResolving.kt

```diff
@@ -19,7 +18,7 @@ internal open class TypeBase<T>
  * Consider using Kotlin's [typeOf] and [KType] since they can
  * also provide information about nullability.
  */
-@Deprecated("Consider using Kotlin type token instead", ReplaceWith("typeOf()"), level = DeprecationLevel.WARNING)
+@Deprecated("Consider using Kotlin type token instead", ReplaceWith("typeOf()"), level = DeprecationLevel.ERROR)
```

Comment:
Spring and/or ktor still use this function, aren't they?

Reply:
They use `serializerByTypeToken` instead